fw/http.c

```diff
@@ -2992,6 +2992,18 @@ tfw_h1_adjust_req(TfwHttpReq *req)
 	int r;
 	TfwHttpMsg *hm = (TfwHttpMsg *)req;
 
+	/* remove X-Tempesta-Cache and change PURGE to GET, if proper flag X-Tempesta-Cache was set */
```

Comment:
This is bit old-fashioned even for the kernel, but usialy we fir 80-characters wide lines, maybe couple of characters more

Reply:
this is draft. I didn't mean this as production code. this comment is for myself.